### ISSUE TITLE
Wrong parameter "friendship_id" for getExpenses

### ISSUE THREAD

**Issue by quaqo** [namaggarwal/splitwise#62]
According to API ref v.3.0 the correct parameter should be "friend_id", friendship_id is then returned for getExpense.

Patch below:

**./splitwise/__init__.py**
```
460c460
<                     friendship_id=None,
---
>                     friend_id=None,
471,472c471,472
<             group_id(long, optional): GroupID of the expenses
<             friendship_id(long, optional): FriendshipID of the expenses
---
>             group_id(long, optional): If provided, only expenses in that group will be returned, and friend_id will be ignored.
>             friend_id(long, optional): ID of another user. If provided, only expenses between the current and provided user will be returned.
```

**Comment by namaggarwal** [comment]
Thanks. Let me have a look

**Comment by CloCkWeRX** [comment]
![image](https://user-images.githubusercontent.com/365751/159150825-027ad993-7e60-43c2-b6ad-9da74e1b182f.png)

**Comment by namaggarwal** [comment]
Fixed in v3.0.0.
